tests/testsuite/git.rs

```diff
@@ -2247,6 +2247,11 @@ fn add_a_git_dep() {
 
     p.cargo("build").run();
 
+    assert!(paths::home().is_dir());
+    assert!(paths::home().join(".cargo").is_dir());
+    assert!(paths::home().join(".cargo/git").is_dir());
+    assert!(paths::home().join(".cargo/git/CACHEDIR.TAG").is_file());
```

Comment:
I guess assert the existence of `CACHEDIR.TAG` is sufficient. Is there any reason to assert intermediate paths?